.github/workflows/commit.yml

```diff
@@ -140,14 +141,15 @@ jobs:
           BRANCH="$(git branch --show-current)"
           git push --atomic origin "${BRANCH}" "${GIT_TAG}"
           echo "git_tag=${GIT_TAG}" >> "${GITHUB_OUTPUT}"
-  cut-release:
-    name: Cut a release
-    needs: [bump-version]
+  distribute:
+    name: Distribute
+    needs: [generate-matrixes, bump-version]
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJSON(needs.generate-matrixes.outputs.image-matrix) }}
+    if: "${{ github.event_name == 'push' && !startsWith(github.event.head_commit.message, 'Bump version: 2') }}"
```

Comment:
This is me being overly cautious. The `needs` section depending on `bump-version` should be enough, but why not.